src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -37,21 +38,26 @@
 import org.apache.commons.lang.ArrayUtils;
 import org.bukkit.event.Event;
 import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.util.SkriptQueue;
 
 import java.lang.reflect.Array;
 import java.util.Iterator;
+import java.util.concurrent.ThreadLocalRandom;
 
 @Name("Elements")
 @Description({
-		"The first, last, range or a random element of a set, e.g. a list variable.",
+		"The first, last, range or a random element of a set, e.g. a list variable, or a queue.",
+	    "Asking for elements from a queue will also remove them from the queue, see the new queue expression for more information.",
 		"See also: <a href='#ExprRandom'>random expression</a>"
 })
 @Examples({
 	"broadcast the first 3 elements of {top players::*}",
 	"set {_last} to last element of {top players::*}",
 	"set {_random player} to random element out of all players",
 	"send 2nd last element of {top players::*} to player",
-	"set {page2::*} to elements from 11 to 20 of {top players::*}"
+	"set {page2::*} to elements from 11 to 20 of {top players::*}",
+	"broadcast the 1st element in {queue}",
+	"broadcast the first 3 elements in {queue}"
 })
 @Since("2.0, 2.7 (relative to last element), 2.8.0 (range of elements)")
```

Comment:
```suggestion
@Since("2.0, 2.7 (relative to last element), 2.8.0 (range of elements), INSERT VERSION (queues)")
```

Reply:
I don't think this is needed because it isn't really a meaningful change to what that syntax does/supports, queues are just another thing that it can work with.